tests/lib/rules/jsx-jcs-no-undef.js

```diff
@@ -147,8 +147,8 @@ ruleTester.run("jsx-jcs-no-undef", rule, {
             parserOptions: {sourceType: "module"}
         },
         {code: "var a; [a] = [0];", parserOptions: {ecmaVersion: 6}},
-        {code: "var a; ({a}) = {};", parserOptions: {ecmaVersion: 6}},
-        {code: "var a; ({b: a}) = {};", parserOptions: {ecmaVersion: 6}},
```

Comment:
I have no idea why old tests had `({b: a}) = {}` there. Which is clearly a syntax error.